pages/desktop/developer_hub/edit_app.py

```diff
@@ -48,20 +48,20 @@ def __init__(self, testsetup):
     def click_edit_basic_info(self):
         self.selenium.find_element(*self._edit_basic_info_locator).click()
         WebDriverWait(self.selenium, self.timeout).until(lambda s: not self.is_element_present(*self._loading_locator)
-            and self.selenium.execute_script('return jQuery.active == 0'))
+                                                         and self.selenium.execute_script('return jQuery.active == 0'))
```

Comment:
Probably out of scope for this but this looks like its worth adding a method to the class, seems to be repeated multiple times.